c++/src/TypeImpl.cc

```diff
@@ -454,6 +454,7 @@ namespace orc {
         throw ParseError("Illegal MAP type that doesn't contain two subtypes");
       if (type.kind() == proto::Type_Kind_UNION && type.subtypes_size() == 0)
         throw ParseError("Illegal UNION type that doesn't contain any subtypes");
+      
```

Comment:
We usually don't mix the bug fix and a style fix in a single PR. Please remove this redundant style change from this PR.